### ISSUE TITLE
Search functionality breaks when source select's options contain hyphenated attributes.

### ISSUE THREAD

**Issue by AndrewDRX** [pytesNET/tail.select#22]
Search functionality breaks when source select's options contain hyphenated attributes.

This issue will most notably occur on data attributes, but can also occur on any other hyphenated attributes (e.g. angularJs ng attributes).

This issue appears to have been introduced w/ commit [9df3441166dbb40c7c98251bf7ab30901b6ed88d] (v0.5.0).

A resolution will likely involve modifications to the regular expression used in the function */js/tail.select.js:finder()*.

Here is a sample used to reproduce:
```
<!doctype html>
<html>
  <head>
    <meta charset="utf-8"> 
    <link rel="stylesheet" href="0.5.3/css/tail.select-default.css"></link>
    <script src="0.5.3/js/tail.select.min.js"></script>
    <script>
      window.onload = function onload() {
        tail.select('#select', {'search': true});
      };
    </script>
  </head>
  <body>
    <div>
      <span>Control:</span>
      <select id="select">
        <option data-test="test 1">
          Test 1
        </option>
        <option data-test="test 2">
          Test 2
        </option>
      </select>
    </div>
  </body>
</html>
```

**Comment by SamBrishes** [comment]
Hello,

thanks for reporting **AND** fixing this issue! Your fix will be released in 0.5.4 (during the next week).

Sincerely,
Sam.

**Comment by SamBrishes** [comment]
Hellow,

Version 0.5.4 has been released just now and contains a Fix for this issue.

Sincerely,
Sam.
